hexadecimal/hex.py

```diff
@@ -0,0 +1,87 @@
+import sys
```

Comment:
Pull Request prefix must contain prefix in ['GENERATOR', 'LEETCODE', 'HEXNUMBER', 'ITERATOR', 'TRIANGLE']